cfme/control/explorer/conditions.py

```diff
@@ -86,6 +86,7 @@ class ConditionFormCommon(ControlExplorerView):
 
 class NewConditionView(ConditionFormCommon):
     add_button = Button("Add")
+    scope_button = Button("Define Scope")
```

Comment:
Let's re-arrange this a bit so that the scope and expression editors click their required buttons before the widget is filled.

Reply:
`ExpressionEditor.show_loc` may actually be functional for this use